vtex-apps/order-manager
# CHK-464: Emit orderForm messages using toast component

#### What problem is this solving?

Emit the orderForm messages using Styleguide's toast component, so the user can be aware of the orderForm messages in any page of the store instead of only in `/cart`.

#### How should this be manually tested?

[Workspace](https://toasts--checkoutio.myvtex.com).

To see the toast, add the first item in the shelf and update its quantity to 51.

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements
